datura.gemspec

```diff
@@ -57,7 +57,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "colorize", "~> 0.8.1"
   spec.add_runtime_dependency "nokogiri", "~> 1.8.1"
   spec.add_runtime_dependency "rest-client", "~> 2.0.2"
-  spec.add_development_dependency "bundler", "~> 1.16"
+  spec.add_development_dependency "bundler", ">= 1.16.0"
```

Comment:
I think this could allow jumps to a 3.x version when written like this. Might want to add another constraint to the end. Looks like it handles that anyway: https://guides.rubygems.org/specification-reference/#add_development_dependency

`spec.add_development_dependency "bundler", ">= 1.16.0", "< 3.0"`?

Reply:
Can do